### ISSUE TITLE
Feature Request: `rgb` Support

### ISSUE THREAD

**Issue by owenthewizard** [LoganDark/stackblur-iter#2]
The [rgb crate](https://crates.io/crates/rgb) is widely used and implementing it in stackblur-iter would make it easier and less error-prone to use than manual conversions.

**Comment by LoganDark** [comment]
This is true, not all u32s are in `0xAARRGGBB` format (and not all buffers are u32s). Could be valuable for those who are doing less bit twiddling.

**Comment by LoganDark** [comment]
I'm going to take a serious look at implementing this now. It looks like it shouldn't be too difficult, but it may end up being a breaking change since I want to reuse the existing `blur_argb` and `blur_srgb` families of functions (to avoid having more than 12(!) top-level functions).

**Comment by LoganDark** [comment]
So `rgb` has a lot of structs, including 16-bit versions. This means `blend-srgb` needs 16-bit support for the sRGB blur functions to support the 16-bit `rgb` types. Everything else can be supported already though.
